data/archetypes/character.json

```diff
@@ -9,7 +9,7 @@
         },
         {
             "name": "anti-hero",
-            "synonyms": [],
+            "synonyms": ["antagonist"],
```

Comment:
An antihero isn't an antagonist, an antihero is a hero who lacks traditional hero-like qualities